admin/src/main/resources/io/buoyant/admin/js/combined_client_graph.js

```diff
@@ -45,10 +45,19 @@ var CombinedClientGraph = (function() {
     var desiredMetrics = _.map(Query.filter(query, metricsCollector.getCurrentMetrics()), clientToMetric);
     chart.setMetrics(desiredMetrics, timeseriesParams, true);
 
-    metricsCollector.registerListener(function(data) {
-      var filteredData = Query.filter(query, data.specific);
-      chart.updateMetrics(filteredData);
-    }, function(metrics) { return Query.filter(query, metrics); });
+    var count = 0;
+    var metricsListener = function(data) {
+      if (count < 5) {
+        // Hacky bug fix: discard the first few data points to fix the issue
```

Comment:
Might be worth opening an issue for investigating why this happens.  Even if we never do it, at least there's a record documenting why we drop data points.